### DISCUSSION TITLE
FLOWS - Webhook Trigger - Authentication

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#17387] [Archive]
Hi - 
does anyone know how I can enable authentication on a webhook trigger in Flows? Currently it seems that ANY unauthenticated requests can trigger a flow... which is fine for some scenarios... but there are a lot of scenarios where authentication is a must....

Thanks,
Roman

**Comment by jonalex** [comment]
I bumped into this as well. 

Has it been resolved.  I don't see a direct authentification request for flows.
